src/components/NFTPortfolio/Item.vue

```diff
@@ -74,8 +77,17 @@ export default {
     };
   },
   computed: {
+    ownedNftId() {
+      return this.collectorMap?.[this.getAddress]?.[0];
+    },
+    isContentViewable() {
+      return this.nftIsNFTBook && this.isBookshelf && this.ownedNftId;
```

Comment:
What if we are viewing the public view of our own portfolio?

Reply:
[🎨 Add current user's portfolio check](https://github.com/likecoin/liker-land/pull/1889/commits/60c5b0430007e9cf10d10d8cdde0eff58a1333f6)